packages/strapi-plugin-users-permissions/controllers/Auth.js

```diff
@@ -142,7 +142,7 @@ module.exports = {
       }
     });
 
-    const provider = ctx.request.url.split('/')[2];
+    const provider = process.platform === 'win32' ? ctx.request.url.split('\\')[2] : ctx.request.url.split('/')[2];
```

Comment:
We could also use the [path.sep](https://nodejs.org/api/path.html#path_path_sep) value?